pkg/domain/entities/images.go

```diff
@@ -306,6 +306,24 @@ type ImageSaveOptions struct {
 	Quiet bool
 }
 
+// ImageScpOptions provide options for securely copying images to podman remote
+type ImageScpOptions struct {
```

Comment:
Podman maintainers, should this be in `pkg/domain/entities` at all?

Reply:
All other image options seem to be in entities, so I feel like it would make sense (especially since I directly use `ImageSaveOptions` and `ImageLoadOptions` in this type) to keep this in here